### PR TITLE
Update the CSP policy

### PR DESCRIPTION
**Pull request by leoapost**
### Trello card
https://trello.com/c/sr8HDii5

### Context
Updating the Content Security Policy improves the overall security of the service

### Changes proposed in this pull request
- Enable nonce only for inline scripts
The [Flipper gem](https://github.com/jnunemaker/flipper/issues/538) and [Google maps](https://stackoverflow.com/a/65451766) do not support nonce for styles yet, so can't enable it for inline styles.

- Fix the font source for the google fonts required by google maps

- Allow connections only to the the [app insights subdomain](https://github.com/microsoft/ApplicationInsights-Ruby/blob/a7429200/lib/application_insights/channel/asynchronous_sender.rb#L20), instead of all the visualstudio.com subdomains

### Guidance to review
Can check the header with curl or from the browser's dev tools

Also can have a run through the pages with the Chrome dev tools console open to see if any related errors are logged.

### CODE REVIEW

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1877-website.azurewebsites.net

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1877-website.azurewebsites.net

File: spec/requests/google_analytics/presence_spec.rb [hunk 1]
```diff
@@ -6,13 +6,14 @@
 
   let(:ga_identifiers) do
     [
-      "<script async src='https://www.google-analytics.com/analytics.js'></script>",
+      "<script src=\"https://www.google-analytics.com/analytics.js\" nonce=\"noncevalue\" async=\"async\"></script>",
```
Comment (peteryates) [inline spec/requests/google_analytics/presence_spec.rb:9]: I know you hate `%(string)` style strings but they have their use! 😂

**Review by peteryates — APPROVED** [review]
Looks good to me.

File: spec/requests/google_analytics/presence_spec.rb (hunk 1 above)
Comment (leoapost) [reply spec/requests/google_analytics/presence_spec.rb]: Don't know why I didn't use single quotes in the first place :sweat_smile: 

There, I just saved my self hundreds of hours from googling what `%` literal I should use for a plain string. :joy:

**Review by peteryates — APPROVED** [review]
Looks good to me

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1877-website.azurewebsites.net

**Pull request merged**